site/resources/faqs.md

```diff
@@ -51,6 +51,23 @@ It is unlikely you will need to frequently update the Light Account contract its
 Yes! The optional index value (salt) on Light Account enables the ability to have multiple accounts for the same owner address. This value defaults to 0. You can set it when you create [light account](/packages/aa-accounts/light-account/).
 :::
 
+### How can I upgrade from Simple Account to Light Account?
```

Comment:
can you make `upgradeToAndCall` as a link to the doc page for the method?
Also, can. you add a mention that devs can also upgrade to Modular Account following the guide "Upgrading to a Modular Account" doc? (hyperlinked to the page)